tests/fixtures/saas/sentry_fixtures.py

```diff
@@ -23,6 +23,11 @@ def sentry_secrets():
         "host": pydash.get(saas_config, "sentry.host") or os.environ.get("SENTRY_HOST"),
         "access_token": pydash.get(saas_config, "sentry.access_token")
         or os.environ.get("SENTRY_ACCESS_TOKEN"),
+        "erasure_access_token": pydash.get(saas_config, "sentry.erasure_access_token") or os.environ.get("SENTRY_ERASURE_TOKEN"),
+        "erasure_identity_email": pydash.get(saas_config, "sentry.erasure_identity_email") or os.environ.get("SENTRY_ERASURE_IDENTITY"),
```

Comment:
I was mainly commenting here that we had to get credentials set up in a separate sentry account because we've been unable to login to the dashboard of the other.  We'd like to consolidate.  

I agree that the erasure identity should be separate from adaptors, we just need to consolidate into one sentry account.

Reply:
The legacy adaptors repo had the concept of a `sar_identity` and an `rtf_identity` with the `rtf_identity` having some unique value appended to it (usually the unix timestamp). Maybe here we can move to the more user-friendly names of `access_identity` and `erasure_identity`. Not for this ticket but something to consider in the future as we standardize our testing.

Reply:
good suggestion